src/index.html

```diff
@@ -17,6 +29,82 @@
     />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header>
+      <a href="#home">
+        <img
+          src="./images/logo.png"
+          alt="moyo logo"
+        />
+      </a>
+      <nav>
+        <ul class="nav_list">
+          <li>
+            <a
+              href="#apple"
+              class="nav-link is-active"
+            >
+              apple
+            </a>
+          </li>
+          <li>
+            <a
+              href="#samsung"
+              class="nav-link"
+            >
+              samsung
+            </a>
+          </li>
+          <li>
+            <a
+              href="#smartphones"
+              class="nav-link"
+            >
+              smartphones
+            </a>
+          </li>
+          <li>
+            <a
+              href="#laptops"
+              class="nav-link"
+              data-qa="hover"
+            >
```

Comment:
The text content of the `<a>` tag is too long and exceeds 80 characters. Consider breaking it into multiple lines for better readability.